common/src/api/external/http_pagination.rs

```diff
@@ -298,15 +296,45 @@ impl ScanParams for ScanById {
 // We include this now primarily to exercise the interface for doing so.
 
 /// Query parameters for pagination by name or id
-pub type PaginatedByNameOrId =
-    PaginationParams<ScanByNameOrId, PageSelectorByNameOrId>;
+pub type PaginatedByNameOrId<Selector> = PaginationParams<
```

Comment:
I think you could do

```suggestion
pub type PaginatedByNameOrId<Selector = ()> = PaginationParams<
```

to avoid changing uses of this that don't require additional selectors

Reply:
done